mix.exs

```diff
@@ -48,6 +48,8 @@ defmodule CircuitsSim.MixProject do
       {:circuits_spi, github: "elixir-circuits/circuits_spi", branch: "configurable-backend"},
       {:circuits_gpio, github: "elixir-circuits/circuits_gpio", branch: "configurable-backend"},
       {:bmp280, "~> 0.2.12", only: [:dev, :test]},
+      {:sgp30, github: "jjcarstens/sgp30", branch: "main", only: [:dev, :test]},
+      {:cerlc, "~> 0.2.1", only: [:dev, :test]},
```

Comment:
`:cerlc` is used in `lib/circuits_sim/device/sgp30.ex`, so it's needed for everything.

```suggestion
      {:cerlc, "~> 0.2.1"},
```